backend/cesium_api/app.py

```diff
@@ -14,11 +15,12 @@ class App(Flask):
 
     def __init__(
         self: App,
-        data: Data,
+        data: Data["DATA"],
         host: str = 'localhost',
         port: int = 9000,
         api_prefix: str = '/',
         data_dir: str = DEFAULT_DATA_DIR,
+        tiles_dir: str = DEFAULT_TILES_DIR,
```

Comment:
```suggestion
```

See related comment below about `DEFAULT_DATA_DIR` vs `DEFAULT_TILES_DIR`.

But make sure to remove this here.